lib/Doctrine/Common/Cache/RedisCache.php

```diff
@@ -89,7 +111,8 @@ protected function doSaveMultiple(array $keysAndValues, $lifetime = 0)
         }
 
         // No lifetime, use MSET
-        return (bool) $this->redis->mset($keysAndValues);
+        $serializedKeysAndValues = array_map([$this, 'serialize'], $keysAndValues);
```

Comment:
This variable isn't necessary

Reply:
You mean $serializedKeysAndValues? It's used on the next line.

Reply:
Yeah. I meant that, since the variable is only used once, we could simply inline the expression. Like:

```php
return (bool) $this->redis->mset(
    array_map([$this, 'serialize'], $keysAndValues)
);
```